drivers/bt_ll_nrfxlib/flash/flash_ll_nrfxlib.c

```diff
@@ -19,13 +19,25 @@
 
 #define SOC_NV_FLASH_NODE DT_INST(0, soc_nv_flash)
 
-/* NOTE: The driver supports unligned writes, but some file systems (like FCB)
+/* NOTE: The driver supports unaligned writes, but some file systems (like FCB)
  * may use the driver sub-optimally as a result. Word aligned writes are faster
  * and require less overhead. This value can be changed to 4 to minimize this
  * overhead.
  */
 #define FLASH_DRIVER_WRITE_BLOCK_SIZE 1
 
+#if DT_NODE_HAS_STATUS(DT_INST(0, nordic_nrf51_flash_controller), okay)
```

Comment:
the nRF Controller is not supported on nrf51 and nrf91. These entries should be removed

Reply:
Thanks. Will remove them.